portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,47 +30,69 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
+/** controls the comments and datastore */
 @WebServlet("/messages")
 public class DataServlet extends HttpServlet {
+
+  /** Adds a comment to datastore */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String title = request.getParameter("comment-box");
+    String comment = request.getParameter("comment-box");
     long timestamp = System.currentTimeMillis();
     String author = request.getParameter("author-box");
+    if(comment.length() > 0){
+      Entity commentEntity = new Entity("Comment");
+      commentEntity.setProperty("comment", comment);
+      commentEntity.setProperty("timestamp", timestamp);
+      commentEntity.setProperty("author", author);
 
-    Entity taskEntity = new Entity("Task");
-    taskEntity.setProperty("comment", title);
-    taskEntity.setProperty("timestamp", timestamp);
-    taskEntity.setProperty("author", author);
-
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.put(taskEntity);
-
+      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Can this be a class member which gets reused on every request?

Reply:
What do you mean by this exactly?